src/spatialCursor/index.js

```diff
@@ -178,18 +177,19 @@ import * as THREE from '../../thirdPartyCode/three/three.module.js';
         onLoadOcclusionObject((worldObject, occlusionObject) => {
             cachedWorldObject = worldObject;
             cachedOcclusionObject = occlusionObject;
+
+            if (DEFAULT_SPATIAL_CURSOR_ON) {
+                toggleDisplaySpatialCursor(true);
+            }
         });
         
         addSpatialCursor();
         addTestSpatialCursor();
-        toggleDisplaySpatialCursor(DEFAULT_SPATIAL_CURSOR_ON);
+        toggleDisplaySpatialCursor(false);
```

Comment:
Is this intentional after toggleDisplaySpatialCursor(true) in onLoadOcclusionObject callback? I got a little confused. Sorry!

Reply:
Since the `onLoadOcclusionObject` includes a 1 second interval before it checks whether the occlusion object is loaded, it seems safe to me to add this after the callback. But it could be more robust if we moved these before it.